lib/feeder_ex.ex

```diff
@@ -22,6 +22,13 @@ defmodule FeederEx do
     :feeder.stream feed_bin, opts
   end
 
+  def parse!(feed_bin) do
+    case :feeder.stream feed_bin, opts do
```

Comment:
I know you already closed this, but i think the canonical way for this would be just to match on ` {:ok, feed, _}`  here and let erlang raise a MatchError if it fails. so it would something like

```
def parse!(feed_bin) do
  {:ok, feed, _} = :feeder.stream(feed_bin, opts)
  feed
end

```